okapi-operation/src/axum_integration/axumyaml.rs

```diff
@@ -0,0 +1,34 @@
+use axum::response::{IntoResponse, Response};
+use bytes::{BufMut, BytesMut};
+use http::{header, HeaderValue, StatusCode};
+use serde::Serialize;
+
+pub struct AxumYaml<T>(pub T);
```

Comment:
To maintain consistency with `Json` type from Axum maybe rename this to `Yaml`? Or at least `YamlResponse`.